src/main/resources/db/migration/V1_18__alter_tables_add_columns.sql

```diff
@@ -0,0 +1,44 @@
+ALTER TABLE court_venue ADD COLUMN welsh_court_name VARCHAR(256);
+ALTER TABLE court_venue ADD COLUMN uprn VARCHAR(16);
+ALTER TABLE court_venue ADD COLUMN venue_ou_code VARCHAR(16);
+ALTER TABLE court_venue ADD COLUMN mrd_building_location_id VARCHAR(16);
+ALTER TABLE court_venue ADD COLUMN mrd_venue_id VARCHAR(16);
+ALTER TABLE court_venue ADD COLUMN service_url VARCHAR(1024);
+ALTER TABLE court_venue ADD COLUMN fact_url VARCHAR(1024);
+ALTER TABLE court_venue ADD COLUMN mrd_created_time TIMESTAMP;
+ALTER TABLE court_venue ADD COLUMN mrd_updated_time TIMESTAMP;
+ALTER TABLE court_venue ADD COLUMN mrd_deleted_time TIMESTAMP;
+
+ALTER TABLE building_location ADD COLUMN welsh_building_location_name VARCHAR(256);
+ALTER TABLE building_location ADD COLUMN welsh_address VARCHAR(512);
+ALTER TABLE building_location ADD COLUMN uprn VARCHAR(16);
+ALTER TABLE building_location ADD COLUMN latitude DECIMAL(10,8);
+ALTER TABLE building_location ADD COLUMN longitude DECIMAL(11,8);
+ALTER TABLE building_location ADD COLUMN mrd_building_location_id VARCHAR(16);
+ALTER TABLE building_location ADD COLUMN mrd_created_time TIMESTAMP;
+ALTER TABLE building_location ADD COLUMN mrd_updated_time TIMESTAMP;
+ALTER TABLE building_location ADD COLUMN mrd_deleted_time TIMESTAMP;
+
+ALTER TABLE court_type ADD COLUMN MRD_Created_Time TIMESTAMP;
```

Comment:
Can we have `MRD_Created_Time` in lower case like above? Please change for other columns as well to be lower case

Reply:
@LukeHalsall please see ^

Reply:
just made the change!